content/en/platform/corda/5.0/application-networks/managing/member-suspension/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+date: '2023-05-09'
+title: "Member Suspension"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-networks-member-suspension
+    parent: corda5-networks-manage
+    weight: 1000
+section_menu: corda5
+---
+
+It is possible to temporarily suspend a member of a group. Once a member has been suspended, Corda blocks flow communication between it and other members of the group. A suspended member performing a member lookup can only see see updates from the MGM and not from other members. It is not possible to suspend the MGM. This section describes how to perform the following:
+* [Query for Members]({{< relref "#query-for-members">}})
```

Comment:
```suggestion
* [Query for Members]({{< relref "#query-for-members">}})
```

Would 'search' sound better than 'query for' in this example - but if that's the internal language being used, better to stick with it.